crates/matrix-sdk-base/src/rooms/mod.rs

```diff
@@ -253,6 +259,11 @@ impl BaseRoomInfo {
                 // wont have call information.
                 return false;
             }
+            AnyStrippedStateEvent::RoomPinnedEvents(p) => {
+                if let Some(pinned) = p.content.pinned.clone() {
+                    self.pinned_events = Some(RoomPinnedEventsEventContent::new(pinned));
+                }
+            }
```

Comment:
I'm not entirely sure about this part.

Reply:
Stripped events = for a room in which we're invited. I think your code is fine, we could also decide to store `None` there, but this shouldn't matter much :+1: I don't expect one to be able to look at pinned events in an invited room, at least for a first pass.